lib/storage/storage_registrar.rb

```diff
@@ -0,0 +1,45 @@
+class StorageRegistrar
```

Comment:
Oh i've seen this Class somewhere, ah it the **CognitoChooser**; oh no it's the StorageRegister